experimental/devices/mfrc522/blockaccess.go

```diff
@@ -0,0 +1,73 @@
+package mfrc522
+
+import "fmt"
+
+// BlockAccess defines the block access bits.
+type BlockAccess byte
+
+// SectorTrailerAccess defines the sector trailing block access bits.
+type SectorTrailerAccess byte
+
+// Access bits.
+const (
+	AnyKeyRWID    BlockAccess = iota
+	RAB_WN_IN_DN              = 0x02 // Read (A|B), Write (None), Increment (None), Decrement(None)
+	RAB_WB_IN_DN              = 0x04
+	RAB_WB_IB_DAB             = 0x06
```

Comment:
Could the name be a bit more self-standing? Right now I have no clue what to make of these constants.

I mean line 14 is a hint, but it's not super clear. Is it a bitmask or individual values?

Reply:
I tried to add some more or less meaningful explanations in the comments, please have a look.